docs/sso_mapping_providers.md

```diff
@@ -91,7 +91,13 @@ A custom mapping provider must specify the following methods:
         `None`, the user is prompted to pick their own username. This is only used
         during a user's first login. Once a localpart has been associated with a
         remote user ID (see `get_remote_user_id`) it cannot be updated.
-      - `displayname`: An optional string, the display name for the user.
+      - `confirm_localpart`: A boolean. If a `localpart` string is returned from
```

Comment:
Confirming means "give them a chance to change it"?

Reply:
I've tried to reword in 48fa7ef0e. WDYT?